src/features/metadata-based-view/MetadataQueryAPIHelper.js

```diff
@@ -107,8 +104,8 @@ export default class MetadataQueryAPIHelper {
         };
     };
 
-    flattenResponseEntry = ({ item, metadata }: MetadataQueryResponseEntry): BoxItem => {
-        const { id, name, size } = item;
+    flattenResponseEntry = (metadataEntry: BoxItem): BoxItem => {
+        const { id, name, size, metadata } = metadataEntry;
```

Comment:
The `/metadata_queries` response structure has changed.
Old response:
```
const response = {
    entries: [
        { item: {...}, metadata: {...} },
        ...
    ],
};
```

New Response:
```
const response = {
    entries: [
        { id: 1, size: 100, metadata: {...} },
        ...
    ],
};
```